api/tests/integration/infrastructure/repositories/session-repository_test.js

```diff
@@ -584,6 +584,33 @@ describe('Integration | Repository | Session', function() {
         });
       });
 
+      context('when there is a filter on the certificationCenterName', () => {
+        let expectedSession;
+
+        beforeEach(() => {
+          const certificationCenter = databaseBuilder.factory.buildCertificationCenter({ name: 'Université des Laura en Folie !' });
+          expectedSession = databaseBuilder.factory.buildSession({ certificationCenter: certificationCenter.name, certificationCenterId: certificationCenter.id });
+          databaseBuilder.factory.buildSession();
+
+          return databaseBuilder.commit();
+        });
+
+        it('should apply a LIKE %% case insensitive type of filter and return the appropriate results', async () => {
```

Comment:
Proposition : avoir une description plus what que how, par exemple "it should find sessions by part of their certification center name, in a case-insensitive way" ?

Reply:
C'est fait, merci